factcast-schema-registry-cli/src/main/resources/site/hugo-theme-docdock-master/exampleSite/content/shortcode/attachments.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "attachments"
+description = "The Attachments shortcode displays a list of files attached to a page."
+
+[menu.main]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4955738)